RELEASE-NOTES.txt

```diff
@@ -1,6 +1,8 @@
 13.1
 -----
- 
+* Added ability to switch between Desktop adn Mobile versions of the content for Site and Page preview.
```

Comment:
```suggestion
* Added ability to switch between Desktop and Mobile versions of the content for Site and Page preview.
```

There is a typographical error in the release notes; ***adn*** should be ***and***.  I'm fine with merging this as is since I believe Editorial will review the release notes.  Let me know what you prefer.

Reply:
Good catch! Fixed it :)